lib/puppet/util/windows/adsi.rb

```diff
@@ -353,7 +354,7 @@ def self.each(&block)
 
       users = []
       wql.each do |u|
-        users << new(u.name)
+        users << new(u.name.encode(Encoding::UTF_8))
```

Comment:
Should "name" be capitalized to signify the same as `group.Name`? Perhaps we can provide an override for name and return it as a string. Seems like that may be quite a bit of work though.